src/support.md

```diff
@@ -10,19 +15,19 @@ The API documentation is [also available][api docs].
 
 ### Supported Rust Versions
 
-Twilight for now follows the latest released stable.
+Twilight currently supports the latest stable Rust version.
 
 ### Breaking Changes
 
 Although Twilight aims to design APIs right the first time, that obviously won't
-happen. A lot of care is taken into designing them in an unopinionated way,
-leaving more opinionated concepts to userland libraries.
+happen. A lot of care is taken into designing clear and correct interfaces.
```

Comment:
`A lot of effort is spent designing clear and correct interfaces`, or `Care is taken while designing...`